Doc/library/concurrent.futures.rst

```diff
@@ -147,7 +147,7 @@ And::
    executor.submit(wait_on_future)
 
 
-.. class:: ThreadPoolExecutor(max_workers=None, thread_name_prefix='', initializer=None, initargs=())
+.. class:: ThreadPoolExecutor(max_workers=None, thread_name_prefix='', initializer=None, initargs=(), **ctxkwargs)
```

Comment:
Shouldn't *ctxkwargs* be documented as well?

Reply:
I added a description and found PR #124548 where this argument was added

Reply:
As it's 3.14+, this means that this should not be backported. So a separate PR is easier to handle as well. And we can make it part of gh-124694 instead.